examples/lighting-app/esp32/main/AppTask.cpp

```diff
@@ -161,7 +161,7 @@ void AppTask::UpdateClusterState()
         ESP_LOGE(TAG, "Updating on/off cluster failed: %x", status);
     }
 
-    ESP_LOGE(TAG, "Writting to Current Level cluster");
+    ESP_LOGI(TAG, "Writting to Current Level cluster");
```

Comment:
Typo: Writing (Please check at other places too)

Reply:
Fixed this and one other